marlowe-chain-sync/docs/marlowe-chain-sync.1

```diff
@@ -1,24 +1,24 @@
-.TH chainseekd 1 "18 July, 2022" "version 0.0.0.0"
+.TH marlowe-chain-sync 1 "18 July, 2022" "version 0.0.0.0"
```

Comment:
Go ahead and delete all these manpage documents. The have not been maintained and are not used.

Reply:
Fixed in 95482d61461f42d5f7abe4850836f26e246b94b3. It looks like we only had one file of this type, but I remember that we used to have more.